src/vs/platform/policy/common/policy.ts

```diff
@@ -39,7 +39,7 @@ export abstract class AbstractPolicyService extends Disposable implements IPolic
 		this.policyDefinitions = { ...policyDefinitions, ...this.policyDefinitions };
```

Comment:
Since `policyDefinitions` is spread before `this.policyDefinitions`, what is currently set in `this.policyDefinitions` will override what's passed in to the function.

I don't know if that's expected or not but since it says "update" I figured I would ask.

Reply:
Thanks, good point.  I don't believe it will matter here in practice (these values are hardcoded into the product so this ends up being more a "merge" than an "update").

Reply:
Made https://github.com/microsoft/vscode/issues/245005 to take a closer look at with @sandy081, I could see this having consequences later as we build out more policy featueres